modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift

```diff
@@ -706,8 +707,129 @@ class SampleCodeTest {
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
 
+        // tag::register-model[]
+        let model = ImageClassifierModel()
+        Database.prediction.registerModel(model, withName: "ImageClassifier")
+        // end::register-model[]
+        
+        // tag::predictive-query-value-index[]
+        let input = Expression.dictionary(["photo": Expression.property("photo")])
+        let prediction = Function.prediction(model: "ImageClassifier", input: input)
+        
+        let index = IndexBuilder.valueIndex(items: ValueIndexItem.expression(prediction.property("label")))
+        try database.createIndex(index, withName: "value-index-image-classifier")
+        // end::predictive-query-value-index[]
+        
+        // tag::unregister-model[]
+        Database.prediction.unregisterModel(withName: "ImageClassifier")
+        // end::unregister-model[]
+    }
+    
+    func dontTestPredictiveIndex() throws {
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
+
+        // tag::predictive-query-predictive-index[]
+        let input = Expression.dictionary(["photo": Expression.property("photo")])
+        let prediction = Function.prediction(model: "ImageClassifier", input: input)
+        
+        let index = IndexBuilder.predictiveIndex(model: "ImageClassifier", input: input)
+        try database.createIndex(index, withName: "predictive-index-image-classifier")
+        // end::predictive-query-predictive-index[]
+    }
+    
+    func dontTestPredictiveQuery() throws {
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
+        
+        // tag::predictive-query[]
+        let input = Expression.dictionary(["photo": Expression.property("photo")])
+        let prediction = Function.prediction(model: "ImageClassifier", input: input) // <1>
+        
+        let query = QueryBuilder
+            .select(SelectResult.all())
+            .from(DataSource.database(database))
+            .where(
+                prediction.property("label").equalTo(Expression.string("car"))
+                .and(
+                    prediction.property("probablity")
+                        .greaterThanOrEqualTo(Expression.double(0.8))
+                )
+            )
+        
+        // Run the query.
+        do {
+            let result = try query.execute()
+            print("Number of rows :: \(result.allResults().count)")
+        } catch {
+            fatalError("Error running the query")
+        }
+        // end::predictive-query[]
+    }
+    
+    func dontTestCoreMLPredictiveModel() throws {
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
```

Comment:
unused `database` variable?

Reply:
Removed in https://github.com/couchbase/docs-couchbase-lite/commit/afe57d9d3c6417437fb00a3eb8e59ce1f1c9469f